src/renderers/WebGLRenderer.js

```diff
@@ -3049,6 +3054,7 @@ THREE.WebGLRenderer = function ( parameters ) {
 
 			}
 
+			// in the future, texture will be expected to be a Texture instance
```

Comment:
Unnecessary. And I wouldn't add comments starting with "in the future". Noone will care LOL :).